README.rst

```diff
@@ -1133,6 +1133,16 @@ We can also query on access type using its numeric value, which will seem more n
 >>> from irods.models import DataAccess, DataObject, User
 >>> data_objects_writable = list(session.query(DataObject,DataAccess,User)).filter(User.name=='alice',  DataAccess.type >= MODIFY)
 
+One last note on permissions:  The <session>.permissions.get(...) method produces inconsistent results when the
+parameter 'report_raw_acls' is allowed to default to False.  Specifically, collections will exhibit the "non-raw-ACL" behavior of
+reporting individual member users' permissions as a by-product of group ACLs, whereas data objects will not.
+
+In release v1.1.6, we move to correct this inconsistency by introducing the synonym <session>.acls that acts almost identically 
+like <session>.permissions, the exception being that in <session>.acls.get(...) method uses a default value of code:`True` rather
+than code:`False` for the `report_raw_acls` parameter.
+
+code:`<session.permissions` is now officially deprecated and will be replaced by <session>.acls in code:`v2.0.0`.
```

Comment:
"is now officially deprecated" -> "is deprecated"

Reply:
Fixed it...